server.js

```diff
@@ -16,12 +17,77 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+const ThoughtSchema = new mongoose.Schema({
+	message: {
+		type: String,
+		required: true,
+		unique: true,
+		minlength: [5, 'The message must be at least 5 characters'],
+		maxlength: [140, 'The message must be less than 140 characters'],
+		trim: true,
+	},
+	hearts: {
+		type: Number,
+		default: 0,
+	},
+	createdAt: {
+		type: Date,
+		default: () => new Date(),
+	},
+});
+
+const Thought = mongoose.model('Thought', ThoughtSchema);
+
+// List all endpoints
+app.get('/', (req, res) => {
+	res.send(listEndpoints(app));
```

Comment:
Also very cool! Did not know we could send a list of all endpoints with this line. I'm learning a lot and I'm less than 50 lines in!